TestApp/app/screens/CrashesScreen.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
-import { Image, View, Text, TextInput, Switch, SectionList, TouchableOpacity } from 'react-native';
-import { DialogComponent } from 'react-native-dialog-component';
+import { Image, View, Text, Switch, SectionList, TouchableOpacity } from 'react-native';
+import DialogInput from 'react-native-dialog-input';
```

Comment:
react-native-dialog-component depends on a lib that is broken in release builds when using react native 0.56: https://github.com/magicismight/react-native-root-siblings/issues/29

This lib has the feature we want so replacing.